test/services/events/EventServiceSpec.scala

```diff
@@ -71,6 +71,12 @@ trait EventServiceFixture extends MockitoSugar {
     assert(eventCaptor.getAllValues.toList.forall(_.eventName == eventName))
   }
 
+  def verifyDataStoreEvents(n: Int, eventNames: List[String]): Unit = {
+    val eventCaptor = ArgumentCaptor.forClass(classOf[DataStoreEvent])
+    verify(dataStoreEventHandlerMock, times(n)).handle(eventCaptor.capture)(any[HeaderCarrier], any[RequestHeader])
+    assert(eventNames.forall(eventName => eventCaptor.getAllValues.toList.exists(_.eventName == eventName)))
```

Comment:
- I think assert from Predef is not designed for unit testing. It gives you unclear error message because it just throw an exception on false.
- forall returns true for empty list, even though eventCaptor.getAllValues returns not empty list.